examples/pure/lifecycleTester.re

```diff
@@ -2,7 +2,7 @@
  * Page component.
  */
 module LifecycleTester = {
-  include ReactRe.Component;
+  include ReactRe.Component.Stateful;
```

Comment:
So before, you had to opt into statelessness. Here, you have to opt into *statefulness*. I think that's a better default.